creator-node/package.json

```diff
@@ -21,14 +24,17 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@audius/sdk": "0.0.15",
+    "@bull-board/api": "^4.3.1",
     "@bull-board/express": "4.0.1",
+    "@opentelemetry/api": "^1.2.0",
```

Comment:
did we not have this before? how was it working?

Reply:
I think it was implicitly installed from another package but having it explicitly installed is definitely much nicer